packages/phrases-ui/src/locales/ru.ts

```diff
@@ -0,0 +1,118 @@
+    social_link_email: 'Вы можете привязать другой адрес электронной почты',
+    social_link_phone: 'Вы можете привязать другой номер телефона',
+    social_link_email_or_phone: 'Вы можете привязать другой адрес электронной почты или телефон',
+    social_bind_with_existing: 'Мы нашли связанную учетную запись, вы можете связать ее напрямую.',
+    reset_password: 'Сбросить пароль',
+    reset_password_description:
+      'Введите {{types, list(type: disjunction;)}} от вашей учетной записи, и мы вышлем вам код подтверждения для сброса пароля.',
+    new_password: 'Новый пароль',
+    set_password: 'Задать пароль',
+    password_changed: 'Пароль изменен',
+    no_account: 'Еще не зарегистрированы? ',
+    have_account: 'Уже есть аккаунт?',
+    enter_password: 'Введите пароль',
+    enter_password_for: 'Введите пароль для {{value}}',
+    enter_username: 'Задать имя пользователя',
+    enter_username_description:
+      'Имя пользователя является альтернативой для входа в систему. Имя пользователя должно содержать только буквы, цифры и символы подчеркивания.',
+    link_email: 'Привязать почту',
+    link_phone: 'Привязать номер телефона',
+    link_email_or_phone: 'Привязать почту или номер телефона',
+    link_email_description:
+      'Для дополнительной безопасности, пожалуйста, привяжите свою электронную почту к учетной записи.',
+    link_phone_description:
+      'Для дополнительной безопасности, пожалуйста, привяжите свой номер телефона к учетной записи.',
+    link_email_or_phone_description:
+      'Для дополнительной безопасности, пожалуйста, привяжите свою электронную почту или номер телефона к учетной записи.',
+    continue_with_more_information:
+      'Для дополнительной безопасности, пожалуйста, заполните приведенные ниже данные учетной записи.',
+  },
+  error: {
+    general_required: `Введите {{types, list(type: disjunction;)}}`,
+    general_invalid: `Проверьте {{types, list(type: disjunction;)}}`,
+    username_required: 'Введите имя пользователя',
+    password_required: 'Введите пароль',
+    username_exists: 'Имя пользователя занято',
+    username_should_not_start_with_number: 'Имя пользователя не должно начинаться с цифры',
+    username_invalid_charset:
+      'Имя пользователя должно содержать только буквы, цифры или символы подчеркивания',
+    invalid_email: 'Электронная почта указана неправильно',
+    invalid_phone: 'Номер телефона указан неправильно',
+    password_min_length: 'Пароль должен быть минимум {{min}} символов',
+    passwords_do_not_match: 'Пароли не совпадают. Пожалуйста, попробуйте еще раз.',
+    invalid_passcode: 'Неправильный код подтверждения',
+    invalid_connector_auth: 'Авторизация недействительна',
+    invalid_connector_request: 'Данные коннектора недействительны.',
+    unknown: 'Неизвестная ошибка. Пожалуйста, повторите попытку позднее.',
+    invalid_session: 'Сессия не найдена. Пожалуйста, войдите снова.',
+  },
+};
+
+const ru = Object.freeze({
+  translation,
+});
```

Comment:
Nit-picking: Should import and use a type here to make sure the phrase keys are identical to what we have in English phrase scripts.
```ts
const ru: LocalePhrase = Object.freeze(
  ...
)
```
But don't worry I can make that change on your behalf